app/graphql/graphql_devise/mutations/resend_confirmation.rb

```diff
@@ -0,0 +1,33 @@
+module GraphqlDevise
+  module Mutations
+    class ResendConfirmation < Base
+      argument :email,        String, required: true, prepare: ->(email, _) { email.downcase }
+      argument :redirect_url, String, required: true
+      
+      field :authenticable, User, null: false
+      field :message, String, null: false
+      
+      def resolve(email:, redirect_url:)
+        resource = controller.find_resource(:uid, email)
+
+        if resource
+          yield resource if block_given?
+
+          raise_user_error("Email #{I18n.t('errors.messages.already_confirmed')}") if resource.confirmed?
```

Comment:
The reason why I did this is because it seemed you wanted to fall back on Devise locales. The errors.message.already_confirmed value is: `"was already confirmed, please try signing in"`. I can make this whole thing as a locale entry but I'll have to put it into the `graphql_devise` locale. Would you prefer that instead?

Reply:
yes please, move the entire message to our own locales file. I think the rule for localized messages should be that ANY message we use directly in OUR code, should go in our locales file. Otherwise we depend on the other gems to keep those keys on their locales files, if for any reason a version of those gems changes the messages, it could affect our gem. I'll have to check the rest of the code for the same thing, but for this PR, please make sure any localized message YOU use in the code has an entry in our locales file (the entire message always as some languages might have a completely different way to say something)

Reply:
Yes check the `/app/views/graphql_devise/mailer/reset_password_instructions.html.erb` file because it also relies on Devise's locale file. I can create a seperate PR for that and fix that.

I'll fix mine and update this PR.

Reply:
Actually it looks like both mailers do this. I'll create a PR for both later.